src/editors/gedit.ts

```diff
@@ -0,0 +1,83 @@
+import os from "os";
+import path from "path";
+
+import Editor from "./editor";
+import { CommandExists } from "../lib/command-exists";
+
+const util = require("util");
+const exec = util.promisify(require("child_process").exec);
+
+export default class Gedit extends Editor {
+  private commandExists = new CommandExists();
+
+  public static getName(): string {
+    return "Gedit";
+  }
+
+  public get name(): string {
+    return "Gedit";
+  }
+
+  public get icon(): string {
+    return "";
+  }
+
+  public async isEditorInstalled(): Promise<boolean> {
+    if (await this.isDirectory(this.appDirectory())) return true;
+
+    if (await this.commandExists.exists("brew")) {
```

Comment:
This only works for Mac, it should be platform agnostic. Something like:

```
if (platform == PLATFORM_MAC) {
  ...
}
```

Reply:
done!